lib/faraday/utils.rb

```diff
@@ -54,7 +53,7 @@ def default_params_encoder
     end
 
     def basic_header_from(login, pass)
-      value = Base64.encode64("#{login}:#{pass}")
+      value = ["#{login}:#{pass}"].pack('m')
```

Comment:
For folks reading this code, to teach this reasonable trick, it'd be nice with a code comment, or, a local private method with a clear name. Either way, a sign-post to readers that don't immediately get the connection between Base64 encoding and pack.

```suggestion
      value = ["#{login}:#{pass}"].pack('m') # Base64 encoding 
```